romdisk/uart.asm

```diff
@@ -124,23 +128,66 @@ uartsnd_main:
         ld l, a
         ; HL contains the string to output on the UART
         ; Get its length and save it in BC
-        push hl
-        call strlen
-        push bc
+
+        ; Filename in HL
+        ld b, h
+        ld c, l
```

Comment:
This can be removed if BC is set directly from (hl)

Reply:
#removed